recipes/partialjson/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "partialjson" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91217e19a15049332df534477f56420065ad1729cedee7d8c7433e1d2acc7dca
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python 3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - partialjson
+  requires:
```

Comment:
Can you add a `- python 3.6` under `test`/`requires`, to comply with the new `noarch: python` requirements:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

I'm not sure if the linter is going to ask you to use the `python_min` jinja variables